StreamVideoTests/StreamVideo_Tests.swift

```diff
@@ -143,10 +143,6 @@ final class StreamVideo_Tests: StreamVideoTestCase {
             streamVideo.state.activeCall == nil
                 && streamVideo.state.ringingCall == nil
         }
-
-        // Then
-        XCTAssert(streamVideo.state.ringingCall == nil)
```

Comment:
Why is this removed?

Reply:
I have removed both lines below
```swift
XCTAssert(streamVideo.state.ringingCall == nil)
XCTAssert(streamVideo.state.activeCall == nil)
```

because the `fulfilment` implementation above, it performs an XCTAssertTrue for the block passed to it which in our case is
```swift
streamVideo.state.activeCall == nil && streamVideo.state.ringingCall == nil
``` and thus verifies the same thing as the remove XCTAsserts.